Automattic/wp-calypso
# Hosting Command Palette: Add navigational commands on Calypso

Related to https://github.com/Automattic/dotcom-forge/issues/4550

## Proposed Changes

This PR adds navigational commands to the command palette on `/sites/` page such as:

- `Open site dashboard`
- `Open stats`
- `Register domain`
- `Open activity log`
- `Open backups`
- `View site metrics`
- `Open PHP logs`
- `Open web server logs`
- `Open hosting configuration`
- `Open SFTP/SSH credentials`
- `Open phpMyAdmin` (should open phpMyAdmin in a new window)
- `Manage staging sites`
- `Manage PHP version`
- `Manage cache settings`
- `Manage admin interface style`


## Testing Instructions

* Navigate to the Calypso Live link
* Access `/sites` page
* Press `cmd + k` on Mac or `ctr + k` on Windows
* Observe the command palette to appear
* Confirm that the navigational commands listed above work as expected 

<img width="1294" alt="Screenshot 2023-11-22 at 9 07 41 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/b9b9794e-9cb9-4a3f-9ef0-08cedc80f808">


## Pre-merge Checklist
